pkg/platform/model/recipe.go

```diff
@@ -179,7 +181,9 @@ func FetchRecipe(commitID strfmt.UUID, owner, project string, hostPlatform *stri
 
 		orderBody, _ := json.Marshal(params.Order)
 		serr := resolveSolverError(err)
-		multilog.Error("Solver returned error: %s, order: %s", errs.JoinMessage(err), string(orderBody))
+		if !locale.IsInputError(serr) {
```

Comment:
Are the underlying errors we care about for this story actually input errors?

Reply:
I believe so. The conflict errors are probably user errors (attempting to remove a non-installed package). Forbidden (authentication) also seems like one. I can leave this bit out if you're not comfortable. The ticket says to use one's best judgement, so this is subjective. I can defer to you.

Reply:
Let's try it.. we can always tune it further :)